tsconfig.json

```diff
@@ -4,6 +4,7 @@
   "include": ["src/**/*"],
   "exclude": ["node_modules/*"],
   "compilerOptions": {
+    "esModuleInterop": true,
```

Comment:
Well I don't understand Javascript well enougy to fully know the implications of this change, but it seems to be sensible and if it doesn't break anything then 🤷‍♂️